tox.ini

```diff
@@ -72,3 +73,10 @@ whitelist_externals=find
 commands =
     safety check -r requirements.txt
     safety check -r requirements-dev.txt
+
+[testenv:bandit]
+deps =
+  bandit
+skip_installs = true
+commands =
+  bandit -ll --skip=B322 -r app server.py
```

Comment:
I'd replace this with `pre-commit`'s bandit hook so that we don't have to maintain two instances of the command for future changes.

Reply:
Good thought on needing to maintain 2.  I removed this tox env and just incorporated bandit into the style check.